hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/JavaTypesArrayTest.java

```diff
@@ -0,0 +1,319 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive.types;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.Month;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.reactive.BaseReactiveTest;
+import org.hibernate.reactive.testing.DBSelectionExtension;
+import org.hibernate.type.SqlTypes;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.vertx.junit5.Timeout;
+import io.vertx.junit5.VertxTestContext;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.COCKROACHDB;
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@Timeout(value = 10, timeUnit = MINUTES)
+
+public class JavaTypesArrayTest extends BaseReactiveTest {
+	@RegisterExtension
+	public final DBSelectionExtension skip = DBSelectionExtension.runOnlyFor( POSTGRESQL, COCKROACHDB );
```

Comment:
A note on the reason this only work for Postgres would be nice.
Don't the other databases  have an array type? is it a problem with the Vert.x client?

Reply:
Working through the changes and making progress on other DB types.  JdbcTypeCode, for some reason is less restrictive with PG. Other DB's returned different sql types, so removing/changing the annotations at the moment.  Making progress, so looks like there will be less impact for other DB's.